spec/requests/v0/profile/addresses_spec.rb

```diff
@@ -348,322 +332,296 @@
     end
 
-    after do
-      Flipper.disable(:va_v3_contact_information_service)
-    end
-
-    let(:address) { build(:va_profile_v3_address, vet360_id: user.vet360_id) }
+    describe 'POST /v0/profile/addresses v2' do
+      let(:address) { build(:va_profile_v3_address, vet360_id: user.vet360_id) }
 
-    context 'with a 200 response' do
-      it 'matches the address schema', :aggregate_failures do
-        VCR.use_cassette('va_profile/v2/contact_information/post_address_success') do
-          post('/v0/profile/addresses', params: address.to_json, headers:)
+      context 'with a 200 response' do
+        it 'matches the address schema', :aggregate_failures do
+          VCR.use_cassette('va_profile/v2/contact_information/post_address_success') do
+            post('/v0/profile/addresses', params: address.to_json, headers:)
 
-          expect(response).to have_http_status(:ok)
-          expect(response).to match_response_schema('va_profile/transaction_response')
+            expect(response).to have_http_status(:ok)
+            expect(response).to match_response_schema('va_profile/transaction_response')
+          end
         end
-      end
 
-      it 'matches the address camel-inflected schema', :aggregate_failures do
-        VCR.use_cassette('va_profile/v2/contact_information/post_address_success') do
-          post('/v0/profile/addresses', params: address.to_json, headers: headers_with_camel)
+        it 'matches the address camel-inflected schema', :aggregate_failures do
+          VCR.use_cassette('va_profile/v2/contact_information/post_address_success') do
+            post('/v0/profile/addresses', params: address.to_json, headers: headers_with_camel)
 
-          expect(response).to have_http_status(:ok)
-          expect(response).to match_camelized_response_schema('va_profile/transaction_response')
+            expect(response).to have_http_status(:ok)
+            expect(response).to match_camelized_response_schema('va_profile/transaction_response')
+          end
         end
-      end
 
-      it 'creates a new AsyncTransaction::VAProfile::AddressTransaction db record' do
-        VCR.use_cassette('va_profile/v2/contact_information/post_address_success') do
-          expect do
-            post('/v0/profile/addresses', params: address.to_json, headers:)
-          end.to change(AsyncTransaction::VAProfile::AddressTransaction, :count).from(0).to(1)
+        it 'creates a new AsyncTransaction::VAProfile::AddressTransaction db record' do
+          VCR.use_cassette('va_profile/v2/contact_information/post_address_success') do
+            expect do
+              post('/v0/profile/addresses', params: address.to_json, headers:)
+            end.to change(AsyncTransaction::VAProfile::AddressTransaction, :count).from(0).to(1)
+          end
         end
       end
     end
-  end
-
-  describe 'POST /v0/profile/addresses 400 v2' do
-    before do
-      Flipper.enable(:va_v3_contact_information_service)
```

Comment:
You don't need `allow(Flipper).to receive(:enabled?)...` for this section?

Reply:
@rmtolmach It's on line 378